### ISSUE TITLE
Segmented Button Fails to Display

### ISSUE THREAD

**Issue by whismand** [RocketCommunicationsInc/astro#685]
Attempting to use a segmented button, and it is not displayed at all.
In the html, we have the <mat-drawer-container> with `<rux-segmented-button data="{{tabLabels}}"></rux-segmented-button>` inside it.
The .ts file has `tabLabels = [ {label: 'Timeline'}, {label: 'List View'}, ]` setup for the button to use. From what we can see, this is all the data that is necessary to create the segmented button, but when run, it appears none of the button is created, there's not even a blank area where the button would have been.

**Comment by micahjones13** [comment]
Hey there,

I've created a [codesandbox](https://codesandbox.io/s/exciting-nobel-tklg7e?file=/index.html) trying to repro the issue. Could you take a look and see if that codesandbox is helpful at all? Also, here's a link from our docs that may be of use as well: https://astro-components.netlify.app/?path=/docs/astro-uxds-welcome-javascript--page#complex-data

Please let me know if I can help further!

**Comment by whismand** [comment]
Alright, I used the example code you provided, and I may have narrowed down the issue. I can get the segmented button to appear properly when it is by itself in the html. But once I put it inside a `<mat-drawer-container>` then it doesn't appear.

**Comment by micahjones13** [comment]
Interesting! I'll check it out in that environment and get back to you.

**Comment by micahjones13** [comment]
Okay, I've tried it out in this [codesandbox](https://codesandbox.io/s/rux-seg-button-mat-drawer-2rcxgp?file=/src/app/app.component.html) running angular and our angular component package. I managed to get the segmented button to render. 
Is this helpful, or is my angular environment too dissimilar? The version difference might be effecting it as well, so if this doesn't help let me know what versions of angular and `@astrouxds/angular` package you're on.
